bin/template.mjs

```diff
@@ -0,0 +1,201 @@
+import ora from 'ora'
+import inquirer from 'inquirer'
+import chalk from 'chalk'
+import fetch from 'cross-fetch'
+import { createRequire } from 'module'
+const require = createRequire(import.meta.url)
+const dictionary = require('./request_schema.json')
+
+const AUTH_METHOD = {
+  APIKey: 'API Key',
+  Cred: 'credentials',
+}
+
+const data = {
+  template: {
+    script: {
+      lang: 'mustache',
+      source: `
+{
+  "query": {
+  "bool": {
+    "must": [
+      {{#query}}
+    {
+      "query_string": {
+      "query": "{{query}}",
+        "fields": ["plot", "title", "genre", "actors"]
```

Comment:
change fields to ["*"]

Reply:
Updated temple like in docs 
<img width="788" alt="image" src="https://github.com/elastic/search-application-client/assets/17390745/def160bc-4dd8-4f1d-89c3-feb7b2f8e777">